### PR TITLE
Update README that this repo is retired.

### PR DESCRIPTION
**Pull request by jerboaa**
Same as for 11u repo. Red Hat will discontinue OpenJDK project builds in favor of Eclipse Temurin.

### CODE REVIEW

**Comment by jerboaa** [comment]
Once this is in we'll archive this repository.

File: README.md [hunk 1]
```diff
@@ -1,49 +1,3 @@
...
-### Only Build OpenJDK 8 (without Build Requirements)
-
-If you already have the build requirements for building OpenJDK 8 installed, you can
-use a simpler build script to build OpenJDK 8:
-
-    $ wget -O openjdk8-upstream-binaries-master.tar.gz "https://github.com/AdoptOpenJDK/openjdk8-upstream-binaries/archive/master.tar.gz"
-    $ tar -xf openjdk8-upstream-binaries-master.tar.gz
-    $ cd openjdk8-upstream-binaries-master
-    $ bash build-openjdk8.sh
+Release scripts and builds archive of former OpenJDK Project builds that are discontinued as of the July 2022 CPU release. No more releases will be provided. Please migrate to a different release such as [Eclipse Temurin](https://adoptium.net/temurin/releases/?version=8).
```
Comment (karianna) [inline README.md]:
```suggestion
This repo contains the release scripts and builds archive of former OpenJDK Project builds that were discontinued as of the July 2022 CPU release. No more releases will be provided. Please migrate to a different release such as [Eclipse Temurin](https://adoptium.net/temurin/releases/?version=8).
```

**Review by karianna — APPROVED** [review]
suggested change for readibility.

**Comment by jerboaa** [comment]
@karianna Thanks!

**Pull request merged**